### PR TITLE
engine_setup: cleanup after ARS removal

### PR DESCRIPTION
**Pull request by mnecas**
Removal of ansible-runner-service constants from engine_setup.

### CODE REVIEW

File: packaging/setup/ovirt_engine_setup/engine/constants.py [hunk 1]
```diff
@@ -266,12 +250,6 @@ class FileLocations(object):
         'runner.wsgi',
     )
 
-    HTTPD_CONF_ANSIBLE_RUNNER_SERVICE = os.path.join(
-        DIR_HTTPD,
-        'conf.d',
-        'zz-ansible-runner-service.conf',
-    )
-
```
Comment (mwperina) [inline packaging/setup/ovirt_engine_setup/engine/constants.py:274]: Hmm, shouldn't we remove this file during engine-setup to handle upgrades?

File: packaging/setup/ovirt_engine_setup/engine/constants.py [hunk 2]
```diff
@@ -266,12 +250,6 @@ class FileLocations(object):
         'runner.wsgi',
     )
```
Comment (mwperina) [inline packaging/setup/ovirt_engine_setup/engine/constants.py:251]: HTTPD_RUNNER_WSGI_SCRIPT is also not needed, right?

Comment (mnecas) [reply packaging/setup/ovirt_engine_setup/engine/constants.py:251]: Oh yeah, missed that one.

**Review by mwperina — CHANGES_REQUESTED** [review]
Please fix python linter issues, other than that it looks good to me

**Comment by mnecas** [comment]
Fixed

File: packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/apache/runner.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
+#
+# ovirt-engine-setup -- ovirt engine setup
+#
+# Copyright oVirt Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+
+
+"""Apache ansible-runner plugin."""
```
Comment (mwperina) [inline packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/apache/runner.py:10]: Apache ansible-runner-service cleanup plugin

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**